x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/exploration_query_bar/exploration_query_bar.tsx

```diff
@@ -59,6 +60,31 @@ export const ExplorationQueryBar: FC<ExplorationQueryBarProps> = ({
 
   const searchChangeHandler = (q: Query) => setSearchInput(q);
 
+  const regex = useMemo(() => new RegExp(`${filters?.columnId}\s?:\s?(true|false)`, 'g'), []);
+
+  /**
+   * If a filter option is active in the url set the corresponding options
+   * button to selected mode
+   */
+  useEffect(() => {
+    if (filters !== undefined) {
+      const match: string[] | null = query.query.match(regex);
```

Comment:
is there a better way to parse the `query`? I think you can use `esQuery` or `esKuery` instead

Reply:
Hm, here we just need to check the query string for the filter pattern - I'm not sure we need to parse it.